### DISCUSSION TITLE
Thoughts on moving 'CEX' and 'funds' into an 'institutional' category?

### DISCUSSION THREAD

**Discussion by andrewhong5297** [duneanalytics/spellbook#2809] [Sector: Labels]
I was looking at labels again, and wondering if it's worth putting CEX and funds labels into an 'institutional' category. That way it's easy to query for if an address is institutional or retail with a join and a CASE WHEN.

@hildobby @MSilb7 @soispoke thoughts?

**Comment by hildobby** [comment]
Yeah that's a good idea imo

**Comment by MSilb7** [comment]
Makes sense
